crates/subspace-networking/src/node.rs

```diff
@@ -273,8 +274,28 @@ impl Node {
         self.shared.id
     }
 
-    pub fn online_status_observer(&self) -> watch::Receiver<bool> {
-        self.shared.online_status_observer_rx.clone()
+    pub fn online_status_observer(&self) -> &watch::Receiver<bool> {
+        &self.shared.online_status_observer_rx
+    }
+
+    /// Prints DSN online status changes.
+    pub fn online_status_informer(&self) -> impl Future<Output = ()> {
```

Comment:
We were talking about a function, but why does purely CLI (and CLI only, and not even for all CLI applications) utility need to be a public method on `Node` in the library?

Reply:
Hmm, it was `online_status_informer(node: &Node)` and I couldn't resist adding it as `online_status_informer(&self)`. I'm moving it back to `utils`.